resources/content/articles/en/2020-05-04_11-00-00_mallet-end-to-end-en.md

```diff
@@ -150,10 +120,11 @@ Create a `myContract.sol` file:
     }
     EOF
 
+**2. Compile with `solc`:**
 
-**2. Compile with `solcjs`:**
+To compile with `solc`, you will need to use the Docker command. For this, first install the [Docker Engine](https://docs.docker.com/engine/install/), then run:
 
-    solcjs --bin --abi --base-path . ./myContract.sol
+    $ docker run -v $(pwd):/sources ethereum/solc:0.5.1 -o /sources --bin --abi /sources/myContract.sol
```

Comment:
`$` at the start is correct to indicate this is a shell command, but it hasn't been used for other commands in this page so I'd suggest either:   remove it here OR add it before all other shell commands.

Reply:
removed `$` here